README.md

```diff
@@ -4,71 +4,91 @@ Longhorn Manager
 
 Manager for Longhorn.
 
-## Requirement
+## Requirements
 
-1. Existing Kubernetes Cluster 1.16+.
-2. Make sure `iscsiadm`/`open-iscsi` has been installed on the host.
+- Kubernetes cluster:
```

Comment:
nit: The other way is to refer to the latest doc instead of duplicating the requirements here again.